addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -438,6 +439,37 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
         }
         this._addHiddenField(value, fieldName);
     },
+    /**
+     * Prompts the user to save changes before being redirected
+     * towards an action specified in value
+     */
+    onClickCreateButton: function (previewMode, value, params) {
+        Dialog.confirm(this, _t("Would you like to save before being " +
+            "redirected? Unsaved changes will be discarded."), {
```

Comment:
You cannot cut translatable string like that: the system which extracts them to be translatable won't understand.